back/app/services/survey_results_generator_service.rb

```diff
@@ -49,7 +49,11 @@ def visit_linear_scale(field)
     values = inputs
       .select("custom_field_values->'#{field.key}' as value")
       .where("custom_field_values->'#{field.key}' IS NOT NULL")
-    distribution = Idea.select(:value).from(values).group(:value).order(Arel.sql('COUNT(value) DESC')).count.to_a
+    distribution_from_db = Idea.select(:value).from(values).group(:value).order(value: :desc).count.to_h
```

Comment:
Shouldn't we only select ideas from the native survey context? It seems that all ideas across the whole platform are selected here.

Reply:
Read line 49 :-) The inputs come from outside.

Reply:
Alright, all good then :)